codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "e2e"
```

Comment:
new line

Reply:
yeah, can configure vscode to automatically add the new line on save

Reply:
solved